check.go

```diff
@@ -12,18 +14,20 @@ import (
 
 // Check details.
 type Check struct {
-	Name   string `json:"name,omitempty"`
-	Web    string `json:"web,omitempty"`
-	Shell  string `json:"shell,omitempty"`
-	Match  string `json:"-"`
-	Return int    `json:"-"`
-	Notify string `json:"-"`
-	Alert  string `json:"-"`
-	Tries  int    `json:"-"`
-	Repeat int    `json:"-"`
-	Sleep  int    `json:"-"`
-	Failed bool   `json:"failed" yaml:"-"`
-	Since  string `json:"since,omitempty" yaml:"-"`
+	Name    string            `json:"name,omitempty"`
+	Web     string            `json:"web,omitempty"`
+	Headers map[string]string `json:"headers,omitempty"`
+	Body    string            `json:"body,omitempty"`
```

Comment:
err, let me be clear:
1. we have a [YAML config](https://github.com/chillum/jsonmon/wiki/Configuration) file that is used for settings checks' parameters
2. we have a [JSON status report](https://github.com/chillum/jsonmon/wiki/RESTful-API) for Web UI

so, again, my question is: why do we need `headers` and `body` in JSON at all?